package.json

```diff
@@ -21,6 +21,8 @@
     "lint:fix": "eslint ./src --ext .js,.jsx --fix"
   },
   "devDependencies": {
+    "@material-ui/core": "^4.12.2",
+    "@material-ui/icons": "^4.11.2",
```

Comment:
Usually we have these dependencies, as well as the styled-components, as project dependencies, instead of devDependencies. This will be needed in a production build so it's not a big deal for now, just FYI